includes/class-newspack-popups.php

```diff
@@ -152,6 +153,18 @@ public static function register_meta() {
 			]
 		);
 
+		\register_meta(
+			'post',
+			'trigger_blocks_count',
```

Comment:
```suggestion
			// Not really a "trigger", since this meta applies only to inline prompts. Keeping the "trigger"-based naming for consistency.
			'trigger_blocks_count',
```